src/Notes/Strasnote.Notes.Api/Controllers/NoteController.cs

```diff
@@ -5,57 +5,126 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Strasnote.AppBase.Abstracts;
+using Strasnote.Logging;
 using Strasnote.Notes.Api.Models.Notes;
 using Strasnote.Notes.Data.Abstracts;
-using Strasnote.Util;
+using Swashbuckle.AspNetCore.Annotations;
 
 namespace Strasnote.Notes.Api.Controllers
 {
+	/// <summary>
+	/// Note Controller
+	/// </summary>
 	[Authorize]
 	[ApiController]
 	[Route("[controller]")]
 	public class NoteController : Controller
 	{
-		private readonly IAppContext ctx;
-
 		private readonly INoteRepository notes;
 
-		public NoteController(IAppContext ctx, INoteRepository notes) =>
-			(this.ctx, this.notes) = (ctx, notes);
+		/// <summary>
+		/// Create object
+		/// </summary>
+		/// <param name="ctx">IAppContext</param>
+		/// <param name="log">ILog</param>
+		/// <param name="notes">INoteRepository</param>
+		public NoteController(IAppContext ctx, ILog<NoteController> log, INoteRepository notes) : base(ctx, log) =>
+			this.notes = notes;
 
+		/// <summary>
+		/// Creates a Note.
+		/// </summary>
+		/// <remarks>
+		/// POST /Note
+		/// </remarks>
+		/// <returns>The ID of the new Note</returns>
 		[HttpPost]
-		public Task<long> Create() =>
-			Is.AuthenticatedUser(
-				ctx,
-				userId => notes.CreateAsync(new()
-				{
-					UserId = userId
-				}),
-				() => 0
+		[SwaggerResponse(201, "The note was created.", typeof(long))]
```

Comment:
We can use [ProducesResponse] instead so we're not tied to Swagger/Swashbuckle

Reply:
All done